spec/openapi.yaml

```diff
@@ -120,6 +120,65 @@ paths:
         '500':
           description: An unexpected error occured.
 
+  /tag/metadata:
+    post:
+      operationId: assign-tag-metadata
```

Comment:
nit: other parts of api use highest concept to lowest concept semantics, to this should be

`tag-metadata-udpate`, `TagMetadataUpdateRequest` etc